src/views/studio/MyUploadsView/AssetLine/AssetLine.tsx

```diff
@@ -47,19 +146,43 @@ const AssetLine: React.FC<AssetLineProps> = ({ isLast = false, asset }) => {
   }
 
   return (
-    <FileLineContainer isLast={isLast}>
-      {isLast ? <FileLineLastPoint /> : <FileLinePoint />}
-      <FileStatusContainer>{renderStatusIndicator(asset)}</FileStatusContainer>
-      <FileInfoContainer>
-        <FileInfoType>
-          {isVideo ? <SvgGlyphFileVideo /> : <SvgGlyphFileImage />}
-          <Text variant="body2">{fileTypeText}</Text>
-        </FileInfoType>
-        <Text variant="body2">{dimension}</Text>
-        <Text>{size}</Text>
-      </FileInfoContainer>
-      <StatusMessage variant="subtitle2">{fileStatusMessage}</StatusMessage>
-    </FileLineContainer>
+    <>
+      <MessageDialog
+        title="Different file was selected!"
+        description="We detected that you selected a different file than the one you uploaded previously. Select the same file to continue the upload or make a new transaction (0.2JOY) and upload a new version of file."
```

Comment:
I'd like the copy here to be `We detected that you selected a different file than the one you uploaded previously. Select the same file to continue the upload or edit "your channel"/"the video" to use the new file.`

We should show the correct variant depending on whether it's a video or a channel